src/dmd/cli.d

```diff
@@ -314,7 +314,37 @@ dmd -cov -unittest myprog.d
             "same as -i=-std,-core,-etc,-object"
         ),
         Option("i=[-]<pattern>,[-]<pattern>,...",
-            "include/exclude imported modules whose name matches one of <pattern>"
+            "include/exclude imported modules whose name matches one of <pattern>",
+            q"{$(P Enables "include imports" mode, where the compiler will include imported
+             modules in the compilation, as if they were given on the command line.)
+
+             $(P Determining which modules to include is based on a set of "module patterns".
+             These patterns are optionally included after `-i=` as a comma separated list.
+             A pattern matches any module whose fully qualified name starts with the pattern.
+             The pattern "foo.bar" matches all modules in the `foo.bar` package, such as
+             "foo.bar" itself, "foo.bar.baz" or even "foo.bar.package". Note that each component
```

Comment:
As you escape package names below, I would do it here too for the sake of consistency.

Reply:
I was attempting to escape package names, but not module patterns...there are other places in the help where module patterns are quoted, but I thought that package names themselves should definitely be escaped.  looking at the final generated docs will likely indicate which method looks the best.